libs/language-server/src/lib/builtin-library/stdlib.ts

```diff
@@ -118,6 +136,53 @@ function parseBuiltinValuetypeToJayvee(valuetype: PrimitiveValuetype): string {
   return lines.join('\n');
 }
 
+function parseConstraintMetaInfToJayvee(
+  name: string,
+  metaInf: ConstraintMetaInformation,
+): string {
+  const lines: string[] = [];
+  if (metaInf.docs.description !== undefined) {
+    lines.push(parseAsComment(metaInf.docs.description));
+  }
+  if (metaInf.docs.examples !== undefined) {
+    metaInf.docs.examples.forEach((example, i) => {
+      lines.push('//');
+      lines.push(`// Example ${i + 1}: ${example.description}`);
+      lines.push(parseAsComment(example.code));
+    });
+  }
+
+  lines.push(`builtin constrainttype ${name} {`);
+  lines.push(parseBuiltinConstrainttypeBody(metaInf));
+  lines.push('}');
+
+  return lines.join('\n');
+}
+
+function parseBuiltinConstrainttypeBody(
+  metaInf: ConstraintMetaInformation,
+): string {
+  const bodyLines: string[] = [];
+
+  Object.entries(metaInf.getPropertySpecifications()).forEach(
+    ([propName, propSpecification]) => {
+      const propDoc = propSpecification.docs?.description;
+      if (propDoc !== undefined) {
+        bodyLines.push(parseAsComment(propDoc, 1));
+      }
+      bodyLines.push(
+        `\tproperty ${propName} oftype ${propSpecification.type.getName()} ${
+          propSpecification.defaultValue !== undefined
+            ? `: ${internalValueToString(propSpecification.defaultValue)}`
```

Comment:
This should not parse to `Infinity` but to the actual number, even if it is a magic number. I am pretty sure it just uses Integer MAX_VALUE or something along those lines. Would get rid of the invalid jayvee code.

Reply:
Addressed in [1d6cb98](https://github.com/jvalue/jayvee/pull/444/commits/1d6cb981023f4234a3cd67127d156ebc68ee06f3)